bitnami/postgresql-repmgr/12/debian-11/rootfs/opt/bitnami/scripts/librepmgr.sh

```diff
@@ -796,6 +878,14 @@ repmgr_initialize() {
             repmgr_upgrade_extension
         else
             debug "Skipping repmgr configuration..."
+            debug "Checking if node is marked for promotion "
```

Comment:
Please remove additional whitespace at the end:
```suggestion
            debug "Checking if node is marked for promotion"
```